src/bp/admin/ui/src/workspace/bots/BotItemCompact.tsx

```diff
@@ -45,10 +46,22 @@ const BotItemCompact: FC<Props> = ({
   reloadBot,
   viewLogs
 }) => {
+  const [supportedLanguages, setSupportedLanguages] = useState([])
+  useEffect(() => {
+    const axios = api.getSecured({ useV1: true })
+
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
+    axios.get(`/bots/${bot.id}/mod/nlu/health`).then(response => {
+      setSupportedLanguages(response.data)
+    })
+  }, [])
+
   const botShortLink = `${window.location.origin + window['ROOT_PATH']}/s/${bot.id}`
   const botStudioLink = isChatUser() ? botShortLink : `studio/${bot.id}`
   const nluModuleEnabled = !!loadedModules.find(m => m.name === 'nlu')
   const hasStudioAccess = isOperationAllowed({ resource: 'studio', operation: 'read' })
+  const languages = intersection(bot.languages, supportedLanguages)
+  const isSupportedLanguage = bot.languages.length === languages.length ? true : false
```

Comment:
Personally, I find the naming of this variable kind of weird. Maybe you could change it to something like `botHasUnsupportedLanguages` and reverse the condition?